spec/flows/user_confirmation_spec.rb

```diff
@@ -0,0 +1,73 @@
+require "rails_helper"
+
+RSpec.describe UserConfirmationFlow, type: :model do
+  describe "#perform" do
+    it "confirms the user if there is receivers available" do
+      Settings.put(Settings::REGISTRATIONS_ENABLED, true)
+      create_list(:receiver, 2)
+      user = create(:user, confirmed_at: nil)
+      user_confirmation = UserConfirmationFlow.new(user.confirmation_token)
+
+      user_confirmation.perform
+
+      expect(user_confirmation).to be_successful
+      expect(user.reload).to be_confirmed
+      expect(user.reload.confirmation_token).not_to be
+      expect(Settings.registrations_enabled?).to be
+    end
+
+    it "confirms the user if there is receivers available and closes registrations if no remaining receivers" do
+      Settings.put(Settings::REGISTRATIONS_ENABLED, true)
+      create(:receiver)
+      user = create(:user, confirmed_at: nil)
+      user_confirmation = UserConfirmationFlow.new(user.confirmation_token)
+
+      user_confirmation.perform
+
+      expect(user_confirmation).to be_successful
+      expect(user.reload).to be_confirmed
+      expect(user.reload.confirmation_token).not_to be
```

Comment:
This and line 28 are already tested on the previous spec, maybe we can ignore it?

Reply:
I don't know, I like to test everything to ensure side effects... what do you think?

Reply:
Yes